spec/alter_table_statement_spec.cr

```diff
@@ -12,15 +12,41 @@ describe LuckyMigrator::AlterTableStatement do
       remove :old_field
     end
 
-    statement.should eq <<-SQL
+    built.statements.first.should eq <<-SQL
     ALTER TABLE users
       ADD name text NOT NULL,
       ADD age int NOT NULL,
       ADD completed boolean NOT NULL,
-      ADD joined_at timestamp NOT NULL,
+      ADD joined_at timestamptz NOT NULL,
       ADD amount_paid decimal NOT NULL,
       ADD email text,
       DROP old_field
     SQL
   end
+
+  it "sets default values" do
+    built = LuckyMigrator::AlterTableStatement.new(:users).build do
+      add name : String, default: "name"
+      add email : String?, default: "optional"
+      add age : Int32, default: 1
+      add num : Int64, default: 1
+      add amount_paid : Float, default: 1.0
```

Comment:
I believe this PR also includes helpers for setting type specific options as well. Could you add a field that uses those?

Reply:
Sure, I added the precision and scale options